openfl/federated/task/runner_fe.py

```diff
@@ -24,11 +33,11 @@ def __init__(self, estimator, **kwargs):
         import fastestimator as fe
 
         class ProgressLoader(fe.trace.Trace):
-            def __init__(self, get_progress) -> None:
+            def __init__(self, get_progress: Callable[[], Dict[str, int]]) -> None:
                 super().__init__(mode='train')
                 self.get_progress = get_progress
 
-            def on_begin(self, data) -> None:
+            def on_begin(self, data: Dict[Any, Any]) -> None:
```

Comment:
Since we have not used parameter `data`
```suggestion
            def on_begin(self) -> None:
```